frontend/src/metabase/setup/selectors.ts

```diff
@@ -81,15 +82,20 @@ export const getIsEmailConfigured = (state: State): boolean => {
 export const getSteps = (state: State) => {
   const usageReason = getUsageReason(state);
   const activeStep = getStep(state);
+  const isPaidPlan = state.setup.isPaidPlan;
```

Comment:
I wish we didn't have to save `isPaidPlan` in the redux store, since that's something that's not going to change ever (unless users login to another instance with a different token of course).

However, based on how `getSteps` is used and other selectors that depend on it. I think it might be justifiable to store it in this Redux slice so that we don't have to always pass `isPaidPlan` through the selector when calling in React components.

Reply:
Have you seen the two PRs linked in the description about this?
Especially [the first](https://github.com/metabase/metabase/pull/39103) solves this with not many changes

Reply:
I didn't see the PR when I made that comment because it was created after my comment. But that one looks good to me.